src/compatTest/groovy/io/github/cosmicsilence/scalafix/ScalafixPluginFunctionalTest.groovy

```diff
@@ -1017,6 +1021,17 @@ class BarDummyRule extends SemanticRule("BarDummyRule") {
         return System.getProperty('compat.gradle.version')
     }
 
+
+    private static boolean isScalaVersionSupported(String scalaVersion) {
+        return scalaVersion.startsWith("2.12")
+
+            // https://github.com/cosmicsilence/gradle-scalafix/pull/85#issuecomment-2588144036
+            || scalaVersion.startsWith("2.13") && gradleVersion() >= '6.0'
+
+            // https://docs.gradle.org/7.3/release-notes.html
+            || scalaVersion.startsWith("3") && gradleVersion() >= '7.3'
+    }
+
     private static boolean isScala3Supported() {
```

Comment:
given that `isScalaVersionSupported` covers all cases, could we drop this method ^^^? (need to update the test that currently uses it)